proto/historyservice/request_response.proto

```diff
@@ -84,7 +86,7 @@ message GetMutableStateResponse {
     // TODO: when migrating to gRPC, make this a enum
     // TODO: when migrating to gRPC, unify internal & external representation
     // NOTE: workflowState & workflowStatus are the same as persistence representation (???)
-    int32 workflowState = 16;
+    execution.WorkflowExecutionState workflowState = 16;
```

Comment:
I think you can remove all comments above now.

Reply:
Done!